### ISSUE TITLE
with v2.0.1, the build is failing when the tag already exists

### ISSUE THREAD

**Issue by mpellerin42** [pkgdeps/git-tag-action#7]
I tried `pkgdeps/git-tag-action@2.0.1` to fix the warning about node 12 in GitHub actions, but this version is failing the build when the tag already exists:

```
Run pkgdeps/git-tag-action@v2.0.1
  with:
    github_token: ***
    github_repo: nuclia/frontend
    version: 1.0.18
    git_commit_sha: b5560223b83316a37a56711910b8174a9dac890c
    git_tag_prefix: v
  env:
    APP_IMAGE_NAME: app
    MANAGER_IMAGE_NAME: manager
    CDN_STORAGE: ***
    CONTAINER_REGISTRY: eu.gcr.io/***
    NX_BASE: ***
    NX_HEAD: ***
    PACKAGE_VERSION: 1.0.18
Error: Reference already exists
```

**Comment by azu** [comment]
Can you try latest version/[v2.0.5](https://github.com/pkgdeps/git-tag-action/releases/tag/v2.0.5)?
https://github.com/pkgdeps/git-tag-action/releases/latest

**Comment by mpellerin42** [comment]
I yes! I missed this version caused I first followed the latest version mentionned in https://github.com/marketplace/actions/pkgdeps-git-tag-action (which is referrencing 2.0.1).
sorry about that.

**Comment by mpellerin42** [comment]
Fixed on version 2.0.5 indeed
